app-localize-behavior.html

```diff
@@ -167,6 +179,20 @@
     }
   },
 
+  _languageChanged() {
+    // Clear the message cache.
+    var proto = this.constructor.prototype;
+    proto.__localizationCache.messages = {};
+
+    for (const instance of appLocalizeBehaviorInstances) {
+      // Update the language of all instances to match this one.
+      instance.language = this.language;
```

Comment:
Hmm, this seems unrelated to the problem you're fixing, and slightly way bigger -- this basically makes it impossible to keep elements that use this behaviour use different languages at the same time. I don't think setting a language is that complicated or verbose, so I'd lean towards not doing this (certainly not doing it in this PR)